src/sortStudents.ts

```diff
@@ -1,16 +1,80 @@
-
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = "Name",
+  Surname = "Surname",
+  Age = "Age",
+  Married = "Married",
+  AverageGrade = "AverageGrade",
```

Comment:
If you change them like this, they can be used as keys
```suggestion
  Name = "name",
  Surname = "surname",
  Age = "age",
  Married = "married",
  AverageGrade = "averageGrade",
```